source/administration/production-checklist-operations.txt

```diff
@@ -192,7 +192,7 @@ Linux
    - Configure sufficient file handles (``fs.file-max``), kernel pid limit
      (``kernel.pid_max``), and maximum threads per process
      (``kernel.threads-max``) for your deployment. For large systems,
-     values of 98000, 32768, and 64000 are a good starting point.
+     values of 98000, 64000, and 64000 are a good starting point.
```

Comment:
Uhhh, no?

The idea is that if pid_max is only 32768, then setting the nprocs ulimit to 64000 won't have the desired effect.  To put it another way, the user expectation is that after setting nprocs to 64000, they can have up to 64000 processes/threads on the system.  But this is only possible if pid_max is also >= nprocs, i.e. is >= 64000.  If it is less --- like the 32768 that we recommend here --- then the number of possible processes/threads will cap out less than 64000, which is confusing.

Therefore, we should ensure that we recommend setting pid_max to at least our recommended value for the nprocs ulimit.  Since we recommend nprocs be 64000, we should also recommend pid_max of 64000 --- which is what this patch does.

This is all assuming that the three values listed here correspond to file-max, pid_max and threads-max, respectively.  Feel free to rework things further to make that clearer, if necessary.